DFKPS/DFKPS-IOC-01App/Db/DFKPS_common.db

```diff
@@ -133,3 +133,14 @@ record(bo, "$(P)DISABLE")
   field(VAL, "$(DISABLE=0)")
 }
 
+record(calcout, "$(P)CURR")
+{
+field(DESC, "Convert Raw Val to Current")
+field(CALC, "A * $(FRI)")
```

Comment:
Wouldn't have caught it, it's because I moved it at your suggestion, the FRI was set in the other .cmd files as they were specific, so I will make those alterations to all three files